Sources/Target/Linux/Process.cpp

```diff
@@ -327,13 +327,19 @@ ErrorCode Process::getMemoryRegionInfo(Address const &address,
     uint64_t inode;
     int nread;
 
-    if (std::fgets(buf, sizeof(buf), fp) == nullptr)
-      break;
+    if (::fgets(buf, sizeof(buf), fp) == nullptr) {
```

Comment:
It's not implementation-defined. It supposedly is "deprecated", but we're using `::` everywhere and these two seemed to be outliers so unless we want to change everything, I suppose this way is better.